00_Base/src/services/TokensAdminService.ts

```diff
@@ -108,7 +111,7 @@ export class TokensAdminService {
         if (
           response.status_code === OcpiResponseStatusCode.GenericSuccessCode
         ) {
-          await this.tokenRepository.updateBatchedTokens(response.data);
+          this.updateTokens(response.data);
```

Comment:
Was the removal of the `await` on purpose? It's still an async call right.

Reply:
It was not on purpose. Added back in.